piplmesh/account/backends.py

```
@@ -44,39 +55,9 @@ def authenticate(self, token=None, request=None):
         # Retrieve user's public profile information
         data = urllib.urlopen('https://graph.facebook.com/me?access_token=%s' % access_token)
         fb = json.load(data)
-
-        try:
-            # Check if user profile exists
-            profile = models.UserProfile.objects.get(facebook_id=fb['id'])
-            user = profile.user
-
-            # Update access token
-            profile.token = access_token
-            profile.save()
-
-        except models.UserProfile.DoesNotExist:
-            # User profile doesn't exist, create new user
-            username = fb.get('username', fb['id'])
-            user = auth_models.User.objects.create_user(username=username, email=fb['email'])
-            user.first_name = fb['first_name']
-            user.last_name = fb['last_name']
```

OK. Then leave it as it was. So let it throw exceptions. Will somebody else check/test this. Then do not convert to `get`. Or at least make smart decisions when to do get and when not.